priya/javascript/Assignment6/JsAssign6Q1.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 names here</h4>
+        Name1: <input type="text" id="name1" placeholder="Enter first name ">
+        <button onclick="MyArray()">Enter</button>
+        <p id="result">
+            <p id="demo"></p>
+            <p id="demo1"></p>
+            <p id="demo2"></p>
+        </p>
+        <br>
+        <br>
+        <script language="javascript">
+        function MyArray(){
+            var b,a,result;
```

Comment:
change variable names  in all files like input 1 and input 2 it acn not be a or b.